### DISCUSSION TITLE
Be proud of what you made

### DISCUSSION THREAD

**Discussion by izzycsy** [DeFiCh/app#680] [Show and tell]
#### Date the project went live / Expected date of launch
_Project established on 18/03/2021_

#### Why the project was launched
_What the project is about and pain points addressed for your audience._

#### What's in the project
_Include screenshots, sneak peaks and data on the project._

#### The results of the project 
_Include metrics that were tracked._

#### Feedback from community
_Feedbacks from the community is much appreciated!_

**Comment by Stonygan** [comment]
mydeficha.in - [Introduction](https://github.com/DeFiCh/app/discussions/681) and [Website](http://mydeficha.in/)

**Comment by izzycsy** [comment]
Sharing a list of current Community Projects:

• Dashboard: https://www.cakedefi-review.com/DefiChain

• Wiki: https://defichain-wiki.com/

• Learning: https://defichain-explained.com

• DeFi Portfolio: https://github.com/DeFi-PortfolioManagement/defi-portfolio

• Mobile app: https://github.com/DeFiCh-WalletApp

• Income Dashboard: https://defichain-income.com

• mydeficha.in: https://mydeficha.in/en/

**Comment by Gainto** [comment]
defichain-promo.com: https://defichain-promo.com
